modules/hfs/samples/example.cpp

```diff
@@ -0,0 +1,50 @@
+#include "opencv2/hfs.hpp"
+#include "opencv2/opencv.hpp"
+
+using namespace cv;
+using namespace cv::hfs;
+int main()
+{
+    String str = "data/000";
```

Comment:
This path will be invalid in most cases. Use command line arguments to provide the path to an input image.

Reply:
alright, I see :)